.circleci/config.yml.in

```diff
@@ -455,21 +455,17 @@ jobs:
     resource_class: gpu.small
     environment:
       image_name: "pytorch/manylinux-cuda101"
+      PYTHON_VERSION: << parameters.python_version >>
```

Comment:
Those changes are also present in #2973, and there is some discussion going on with @mthrok on the best way to solve this.

In order to move forward with this PR, I'll be merging this as is but let's maybe revisit this implementation in a follow-up PR following @mthrok and @seemethere feedback.